### PR TITLE
glances 3.1.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/glances.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Glances < Formula
   desc "Alternative to top/htop"
   homepage "https://nicolargo.github.io/glances/"
-  url "https://github.com/nicolargo/glances/archive/v3.1.4.1.tar.gz"
-  sha256 "0347a0b949451fd0022c0f22e54092fe526120a776af1f2bde1ea7ba61d6b792"
+  url "https://github.com/nicolargo/glances/archive/v3.1.5.tar.gz"
+  sha256 "e60ea1274e882f9d64d16c12690bd71dcc70ebe7e93690d773275f92fa126b24"
   license "LGPL-3.0"
```
Comment (SMillerDev) [inline Formula/glances.rb:6]: or later

File: Formula/glances.rb [hunk 2]
```diff
@@ -14,219 +14,14 @@ class Glances < Formula
 
   depends_on "python@3.8"
 
-  resource "bernhard" do
```
Comment (SMillerDev) [inline Formula/glances.rb:17]: I think theses should stay

**Comment by mabbamOG** [comment]
this PR has been blocked for a while now, and the package outdated for some months. Is the main issue an outdated license?

**Comment by SMillerDev** [comment]
> this PR has been blocked for a while now, and the package outdated for some months. Is the main issue an outdated license?

Main issue is no response from @chenrui333 on comments. Where the unexplained removal of resources is a bigger problem then the license.

**Comment by chenrui333** [comment]
sorry, I missed this, picking this up.

**Comment by chenrui333** [comment]
<details>
<summary>license header ref</summary>

```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# Glances - An eye on your system
#
# Copyright (C) 2019 Nicolargo <nicolas@nicolargo.com>
#
# Glances is free software; you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# Glances is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program. If not, see <http://www.gnu.org/licenses/>.
```

</details>

**Review by SMillerDev — DISMISSED** [review]

**Pull request closed without merging**